pkg/catalog/catalog.go

```diff
@@ -87,13 +87,19 @@ func VersionExists(client *occlient.Client, componentType string, componentVersi
 }
 
 // getDefaultBuilderImages returns the default builder images available in the
-// openshift namespace
+// openshift namespace and current namespace
 func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
```

Comment:
This image should also return in what namespace the image is. as there might be two images with the same name in both openshift and current namespace. We also have to figure out what we will do if there is a collision.

Reply:
Ack

The o/p now looks like this: https://gist.github.com/anmolbabu/028a4caf25c009e3c63c82721f8146f3